src/Config.php

```diff
@@ -4,6 +4,12 @@
 
 use Tnapf\Config\Exceptions\InvalidConfigException;
 
+use function is_array;
```

Comment:
To my knowledge these `use`s do absolutely nothing and are not required by PSR/PER code styling. Am I wrong here or can these just be removed?

Reply:
> To my knowledge these `use`s do absolutely nothing and are not required by PSR/PER code styling. Am I wrong here or can these just be removed?

Yes they can be removed, it's just my IDE messing with me again (corporate code style settings)... I'll remove these imports.